d

```diff
@@ -0,0 +1,20 @@
+[33mc01b6cb[m[33m ([m[1;36mHEAD -> [m[1;32mDevelop[m[33m, [m[1;31morigin/Develop[m[33m)[m Revert "Merge pull request #4 from infoshareacademy/bt-7"
```

Comment:
What is this? and what is it for? comments?

Reply:
lmao i have no clue, is this gitlog? no idea why its here